install/jsonp/docs/assertions/JSONP2.0JavadocAssertions.html

```diff
@@ -24,7 +24,7 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>  Jakarta JSON Processing - 1.1<br>
+<h2>  Jakarta JSON Processing - 2.0<br>
```

Comment:
we need to update Copyright (c) 2018, 2019 to "Copyright (c) 2018, 2020"

Reply:
fixed :v:

Reply:
thanks